wasm_utils/wasm-test/integration-test/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "integration-test"
+version = "0.1.0"
+authors = [""]
+
+[lib]
+crate-type = ["cdylib"]
+
+[profile.release]
+panic = "abort"
```

Comment:
what's panic = "abort" do?

Reply:
I don't know. It's the default config we have been using.
I'm actually planning on looking into panic! today.